up42/tools.py

```diff
@@ -71,6 +78,7 @@ def read_vector_file(
 
         if df.crs.to_string() != "EPSG:4326":
             df = df.to_crs(epsg=4326)
+        df.geometry = df.geometry.buffer(0)
```

Comment:
hm this was removed in a recent PR, can you check if this was reintroduced by accident? Seems you are on top of master.

Reply:
Removed!